advanced_functionality/multi_model_tensorflow/tf2-MME-regression.ipynb

```diff
@@ -0,0 +1,1534 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    test = pd.read_csv('/home/ec2-user/SageMaker/TensorFlow-MME/Data/Boston/train.csv')
</pre><p>Don't hard-code absolute paths - the customer may have their data in a different location. (There are 4 instances of this within the notebook.)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2887/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>